docs/design-methodology/Networking.md

```diff
@@ -520,11 +520,13 @@ The delivery of network policies is consistent regardless of whether Azure or Ca
     - This will ensure traffic flows through Azure Front Door at a service level, but header based filtering will still be required to ensure the use of a particular Front Door instance and to also mitigate 'IP spoofing' security risks.
   - Public internet traffic should be disabled on RDP and SSH ports across all applicable NSGs.
 
+- Decide between Azure CNI and Kubenet at deployment time based on requirements and capabilities like the amount of IP addresses consumed, the connectivity requirements of the application, and other aspects.
+  - For example, if the application requires a high amount of IP addresses, consider using Kubenet.
```

Comment:
This is a duplicate recommendation we have the same on L356-357 already.

Reply:
modified it and linked to the k8s section for more details